.github/workflows/deploy-website.yaml

```diff
@@ -2,7 +2,7 @@ name: Update `error-prone.picnic.tech` website content
 on:
   pull_request:
   push:
-    branches: [ master ]
+    branches: [ master, develop ]
```

Comment:
Why do we need `develop`? That branch didn't exist before.

Reply:
Below we're mentioning `website`, so perhaps it should be `s/develop/website/`?

Reply:
You are right, not sure why I typed `develop` 😅. Pushed the tweak.

Reply:
I think we can omit this, as long as you open a draft PR. Would that work? Or is there a strong preference to keep using  a branch?

Reply:
For now it would be nice to be able to quickly push something to that branch, such that it'll build. Making draft PRs is a bit more cumbersome. We can revert this after we have everything stable and merged the way we want it. 

For now it'll allow us to iterate quickly.

Reply:
In that case, we don't even need to merge this PR. Just push this change on the `website` branch, and it'll work :)